src/problem-analysis/problem-statement.tex

```diff
@@ -0,0 +1,25 @@
+% textidote: ignore begin
+\section{Problem statement}\label{sec:problem-statement}
+% textidote: ignore end
+When meeting the owners of the Caf\'e, they established that they wanted to have a system that could provide them with
+an overview of their sales, so they could make better decisions based on their data.
+The data provided by the system would help optimize staff shifts, ensuring that the right number of employees are
+scheduled for the right time of day.
+
+Additionally, the caf\'e sells baked goods that need to be prepared in advance.
+The system should enable the owners to analyze the sales data provided to see the customer traffic from previous weeks,
+ensuring they can prepare the right amount of baked goods for the upcoming days.
+
+With one of the owners not being very tech-savvy, we would need to make sure that the system is easy to use and
+understand, so it is critical that we focus on the UX of the system.
+
+This presents us with the following problem statement:
+%textidote: ignore begin
```

Comment:
What if textidote suggest that the first letter needs to be uppercase. Adding "tcolorbox" to the whitelist file does not resolve that. Should I just leave the last ignore on line 21 because that is where this problem occurs.

Reply:
Then leave it